src/content/docs/synthetics/synthetic-monitoring/getting-started/new-runtime.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: Convert to the new synthetic runtimes
+tags:
+  - Synthetics
+  - Synthetic monitoring
+  - Runtime conversion
+metaDescription: "Synthetic monitoring's new runtime requires that you make some changes to your already existing monitors."
+---
+
+We're introducing a new runtime for public monitoring locations. We plan to apply the new runtime to private monitors by the end of 2022. While we're providing backwards support for monitors using the Legacy runtime and its features, we recommend converting to the current runtime as soon as possible. 
+
+Going forward, new monitors are defaulting to the new runtime while monitors created using the Legacy version will need additional configuration.
```

Comment:
I think you can simplify this a bit too, because it's important:

"New synthetic monitors default to using the new runtime (a name other than "new runtime" would be great!). If you create a synthetic monitor with the legacy runtime, there are extra configuration steps (add link to extra config steps)."